test/runtests.jl

```diff
@@ -1,6 +1,7 @@
 using AstronomicalTime
 using Base.Test
-import ERFA: eraDat
+using ERFA
+include("../src/conversions.jl")
```

Comment:
Usually you shouldn't include a file from source directory (and definitely don't do it in this way).  We're loading `AstronomicalTime` module above, everything is already available.  What's the purpose of including this file?